src/components/Content/Card.jsx

```diff
@@ -33,30 +43,37 @@ const Card = ({ id, name, bookmark, poster_path, title, release_date, first_air_
 
       <Bookmark
         className={styles.bookmark}
-        id={id} media_type={media_type} key={id} bookmark={bookmark}></Bookmark>
+        id={id}
+        media_type={media_type}
+        key={id}
+        bookmark={bookmark}
+      ></Bookmark>
 
       {/* Content section below the image */}
       <div className="content-sec px-2 py-1 text-white text-xs">
         <ul className="flex content-sec gap-x-3 mt-2">
-        <li className="flex flex-col items-center text-white text-xs">
-        <span className="mr-1">
-            { release_date?release_date.slice(0,4):first_air_date.slice(0,4)}
-        </span>
-        </li>
-        
-        <li className="flex items-center">
-          <RiFilmFill className="mr-1 text-white" />
-          <span>{media_type}</span>
-        </li>
-        <li className="flex items-center text-xs">
-          <span>{adult ? "PG" : "UG"}</span>
-        </li>
-      </ul>
-      <h3 className="mt-2 line-clamp-1 text-lg tracking-tight">
-        {title?title:name}</h3>
+          <li className="flex flex-col items-center text-white text-xs">
+            <span className="mr-1">
+              {release_date?.slice(0, 4) ||
```

Comment:
We are banning ternary operator for better readibility.
Hence refactor it to a function.